src/main/java/org/broadinstitute/hellbender/tools/spark/sv/BreakpointAlignment.java

```diff
@@ -0,0 +1,170 @@
+     * Assumes {@code region1} has a lower {@link AlignmentRegion#startInAssembledContig} than {@code region2}.
+     */
+    public BreakpointAlignment(final String contigId, final AlignmentRegion region1, final AlignmentRegion region2, final String insertedSequence, final String homology, final List<String> insertionMappings) {
+        this.contigId = contigId;
+        this.region1 = region1;
+        this.region2 = region2;
+        this.insertedSequence = insertedSequence;
+        this.homology = homology;
+        this.insertionMappings = insertionMappings;
+    }
+
+    @Override
+    public String toString() {
+        return contigId +
+                "\t" +
+                region1.toString() +
+                "\t" +
+                region2.toString() +
+                "\t" +
+                ("".equals(insertedSequence) ? NO_SEQUENCE : insertedSequence) +
+                "\t" +
+                ("".equals(homology) ? NO_SEQUENCE : homology);
+    }
+
+    /**
+     *  Parses a tab-delimited assembled breakpoint line into an BreakpointAlignment object. Fields should be in the same order as that produced by toString():
+     *
+     *  contigId
+     *  alignmentRegion1.toString()
+     *  alignmentRegion2.toString()
+     *  insertedSequence
+     *  homology
+     *
+     */
+    public static BreakpointAlignment fromString(String assembledBreakpointLine) {
+        final String[] fields = assembledBreakpointLine.split("\t");
+        return fromFields(fields);
+    }
+
+    private static BreakpointAlignment fromFields(final String[] fields) {
+        try {
+            final String contigId = fields[0].replaceFirst("^>","");
+            final String[] alignmentRegion1Fields = Arrays.copyOfRange(fields, 1, 10);
+            final AlignmentRegion alignmentRegion1 = AlignmentRegion.fromString(alignmentRegion1Fields);
+            final String[] alignmentRegion2Fields = Arrays.copyOfRange(fields, 10, 19);
+            final AlignmentRegion alignmentRegion2 = AlignmentRegion.fromString(alignmentRegion2Fields);
+            final String insertedSequence = fields[19].equals(NO_SEQUENCE) ? "" : fields[19];
+            final String homology = fields[20].equals(NO_SEQUENCE) ? "" : fields[20];
+            final List<String> insertionMappings = Arrays.asList(fields[21].split(";"));
+            return new BreakpointAlignment(contigId, alignmentRegion1, alignmentRegion2, insertedSequence, homology, insertionMappings);
+        } catch (final NumberFormatException nfe) {
+            throw new GATKException(Arrays.toString(fields), nfe);
+        }
+
+    }
+
+    private AlignmentRegion getLeftAlignmentRegion() {
+        if (region1.referenceInterval.getStart() < region2.referenceInterval.getStart()) {
+            return region1;
+        } else {
+            return region2;
+        }
+    }
+
+    @VisibleForTesting
+    SimpleInterval getLeftAlignedLeftBreakpointOnReference() {
```

Comment:
I've refactored these two methods into a single method called `getLeftAndRightBreakpointsOnReferenceLeftAlignedForHomology`. 

I don't think that the word "Parsimony" really is relevant to what the logic is doing here, or that it describes the process -- left aligning for homology is just done for the purpose of getting a canonical breakpoint location for each pair of alignments. 

I left the method `getLeftAlignmentRegion` as is because I think it makes it explicit what the check is in terms of the naming scheme of the other variables.